src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` for initialization if you need a fresh instance of `RegistrationServiceImpl` before each test. `@BeforeAll` should be used when you want to initialize something once for all tests, such as loading heavy resources.